### DISCUSSION TITLE
Cool things I can do with unpacked extension?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#956] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Found a cool bug where i can get unpacked/extensions from chrome web store thats not blacklisted on my school accounts, any ideas?
Preferred is smth like incognito where i dont get autoblocked from websites like proxies do
Disabling goguardian is fine though
Thanks in advance
also
if anyone here knows php and wants to help me with some code, dm me on discord IFWI4 please im dying

**Comment by v4pl** [comment, marked as answer]
How are you able to disable goguardain?
